apps/dashr-web-trader/app.R

```diff
@@ -261,15 +263,16 @@ app$callback(
       ifelse(
         length(
           charts[!is.na(charts)]) == 4,
-          s["height"] <- "50%",
-          s["height"] <- "100%"
+        s["height"] <- "50%",
+        s["height"] <- "100%"
```

Comment:
:eyes: Yeah the indentation is definitely not ideal here. Maybe even clearer would be:

```
    ifelse(
      length(charts[!is.na(charts)]) == 4,
      s["height"] <- "50%",
      s["height"] <- "100%"
    )
```